examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

```diff
@@ -24,21 +33,42 @@
      */
     @Bean
     public ArmeriaServerConfigurator armeriaServerConfigurator() {
-        // An example which adds a LoggingService decorator to the ServerBuilder in order to log every request.
-        return builder -> builder.decorator(LoggingService.newDecorator())
-                                 .accessLogWriter(AccessLogWriter.combined(), false);
+        // Customize the server using the given ServerBuilder. For example:
+        return builder -> {
+            // Add DocService that enables you to send Thrift and gRPC requests from web browser.
+            builder.serviceUnder("/docs", new DocService());
+
+            // Logging every request and response which is received by the server.
```

Comment:
`Logging` -> `Log` because we used `add` and `write`?

`which are received`? or `which the server receives.`?